src/storm-pars/modelchecker/region/SparseMdpParameterLiftingModelChecker.cpp

```diff
@@ -45,7 +45,7 @@ namespace storm {
         }
 
         template <typename SparseModelType, typename ConstantType>
-        void SparseMdpParameterLiftingModelChecker<SparseModelType, ConstantType>::specify(Environment const& env, std::shared_ptr<storm::models::ModelBase> parametricModel, CheckTask<storm::logic::Formula, typename SparseModelType::ValueType> const& checkTask, bool generateRegionSplitEstimates, bool allowModelSimplifications) {
+        void SparseMdpParameterLiftingModelChecker<SparseModelType, ConstantType>::specify(Environment const& env, std::shared_ptr<storm::models::ModelBase> parametricModel, CheckTask<storm::logic::Formula, typename SparseModelType::ValueType> const& checkTask, [[maybe_unused]] bool generateRegionSplitEstimates, bool allowModelSimplifications) {
```

Comment:
I guess this should not be unused? @jipspel @tquatmann

Reply:
Iirc, this has not been implemented for pMDPs. I suggest a warning:
```suggestion
        void SparseMdpParameterLiftingModelChecker<SparseModelType, ConstantType>::specify(Environment const& env, std::shared_ptr<storm::models::ModelBase> parametricModel, CheckTask<storm::logic::Formula, typename SparseModelType::ValueType> const& checkTask, bool generateRegionSplitEstimates, bool allowModelSimplifications) {
        STORM_LOG_WARN_COND(!generateRegionSplitEstimates, "Requested region split estimates which is not supported for pMDPs");
```
We could also `STORM_LOG_THROW` here, but I don't know where this is actually used. Maybe this would break something in prophesy?